docs/howtos/ckeditor.md

```diff
@@ -197,3 +197,17 @@ apos.define('apostrophe-rich-text-widgets-editor', {
 > If we want to, we can look at `self.options.templateOptions`, which contains the configuration passed to this widget by `apos.areas` or `apos.singleton`.
 
 > Apostrophe's initial definition of the `beforeCkeditorInline` method is empty (following the [template pattern](https://en.wikipedia.org/wiki/Template_method_pattern)), but if you are using various add-on modules it's possible that some of them define it. If you want to be sure that code is called too, use the [super pattern](/reference/glossary.md#super-pattern) rather than just replacing the method outright.
+
+## Displaying icons for ckeditor plugins distributed with ckeditor
```

Comment:
I think this should be a third level (h3) child of `## Global CKeditor configuration`.

Reply:
I'd be fine with that! Curious if Rakhi has anything to add, but we should probably get this out there and integrate feedback when we receive it.